src/core/grpc_service.proto

```diff
@@ -250,6 +256,100 @@ message ControlResponse
   RequestStatus request_status = 1;
 }
 
+//@@.. cpp:var:: message InferSharedMemory
+//@@
+//@@   The meta-data for the shared memory from which to read the input
+//@@   data and/or write the output data.
+//@@
+message InferSharedMemory
```

Comment:
Perhaps InferSharedMemoryRegion. Need a "string name" field so that the user can give a name to the region that they then refer to in their input/output requests.